.github/workflows/rust.yml

```diff
@@ -100,16 +100,22 @@ jobs:
       if: ${{ matrix.test && matrix.os != 'windows-latest' }}
       run: |
         set -euxo pipefail
-        for bin in $bins; do ./${{ matrix.output }} binstall --no-confirm $bin; done
-        ./${{ matrix.output }} binstall --manifest-path . --no-confirm cargo-binstall
+        for bin in $bins; do ./${{ matrix.output }} binstall --log-level debug --no-confirm $bin; done
+        for bin in $test_bins; do $HOME/.cargo/bin/$bin --version; done
+        cargo binstall --help >/dev/null
+        ./${{ matrix.output }} binstall --log-level debug --manifest-path . --no-confirm cargo-binstall
+        cargo binstall --help >/dev/null
       env:
         bins: cargo-deb cargo-llvm-cov cargo-binstall
+        test_bins: cargo-deb cargo-llvm-cov
```

Comment:
out of scope for this PR: I'm a bit uncomfortable depending on external crates for our own testing. I'm not sure how to resolve that yet, though.

Reply:
These crates are available on quickinstall, so it's unlikely they are going to have random breaks.